src/MonitoringFunctions.Linux/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Is there reason, why is this not part of main .gitignore?

Reply:
I am not sure as these are auto-generated. But it maybe because different projects can have different ignore files and it is easier to maintain separate files instead of merging them all.